fluidBuild.config.cjs

```diff
@@ -293,19 +300,17 @@ module.exports = {
 			"npm-package-exports-apis-linted": [
 				// Rollout suppressions - enable only after tools are updated to support policy
 				// as new build-tools will have the concurrently fluid-build support it uses.
-				"^azure/",
 				"^common/",
-				"^examples/",
-				"^experimental/",
-				"^packages/",
 
 				// Packages that violate the API linting rules
-				// AB#8135: ae-unresolved-inheritdoc-reference: @public AzureMember references @internal AzureUser
-				"^packages/service-clients/azure-client/",
 				// ae-missing-release-tags, ae-incompatible-release-tags
 				"^examples/data-objects/table-document/",
 				// AB#8147: ./test/EditLog export should be ./internal/... or tagged for support
 				"^experimental/dds/tree/",
+				// AB#8287 "TokenResponse" is @beta, but used by "TokenFetcher" which is /legacy.
```

Comment:
We're still using `@alpha` tags to mean "legacy", right? I think we need to have a larger conversation about this. `@alpha` items can reference `@beta` items. I don't think we want to mess with that invariant. Until we've properly separated `@alpha` from `@legacy`, we probably just need to apply the same support guarantees that we make for "legacy" exports in `@beta` exports. Then we don't have to worry about this. Those invariants get cleaned up once the "legacy" and "alpha" concepts are separated.

@anthony-murphy Since we were discussing this earlier.

Reply:
This is step one (test/lint) before being able to switch to use `@legacy` instead of `@alpha`.
This linting gives us the ability to not make the same `@beta` support contract as `@alpha-that-means-legacy`, hence the bug :)
Step 2 (I think): tag all `@alpha` with `@legacy` as well (except devtools which is already separable)
Step 3: update rollups to generate /legacy from `@legacy` tags (should also start generating foo.legacy.api.md reports)
   `@alpha` is now reclaimed for traditional use!
Step 4: update `@legacy` tagged APIs surfaced in `fluid-framework` to be `@public` instead of `@alpha`.

Reply:
I'm still not understanding the bug. Why can a `@legacy + @alpha` API not reference a `@beta` API? Or is the rule more nuanced, requiring references to either be `@public` or `@legacy + <appropriate equal-or-stricter release variant>` .

Reply:
`@legacy` is unstable and `@beta` is unstable, but not the same unstable. (`@legacy+@public` is stable.)
`@legacy+@alpha` is relatively unstable and has rules. `@beta` is breakable across minor updates.
Think of `@legacy` as its own space. The `@public|beta|alpha` attached is subdivision. (I would prefer not to have `+@beta|alpha` with `@legacy` but api-extractor wants something.)
`@legacy+@beta` is not anything I think we ever try to have or at least not anytime soon. I could see if someone wanted to say `@legacy+@beta` is more stable than `@legacy+@alpha` but that would be within the world of `@legacy`.

Reply:
I think the two spaces are orthogonal. I can see a reason for `@beta + @legacy` existing because we may wish to say that something is `@legacy` in terms of our legacy/MS-internal support commitments, but `@beta` in terms of how external users should reason about it. And since types can be shared between our many divergent API surfaces, that gives us a consistent way to reason about support contracts that doesn't require us to create lots of duplicate types just to support different contracts. IMO

Reply:
I think a reasonable rule would be that legacy APIs must only reference public APIs or other legacy APIs of the same release level or higher. It's a derivation of the standard rule that APIs must only reference APIs of the same release level or higher, constrained to the "legacy" space.

Reply:
> I think a reasonable rule would be that legacy APIs must only reference public APIs or other legacy APIs of the same release level or higher. It's a derivation of the standard rule that APIs must only reference APIs of the same release level or higher, constrained to the "legacy" space.

That is the rule - just better said than what I wrote. :) (At least I think we agree.)
1. Anything can use `@public`.
2. `@alpha` can use `@beta`.
3. `@legacy` is a self-contained microcosm of general rules.

Reply:
I just want to make sure everyone is on the same page, and the correct resolution to any issues we identify is clear (until tooling enforcement is in place).

Reply:
Well, you said "TokenResponse is `@legacy`" but that is not correct. Were you trying to say "TokenResponse is not `@legacy` and not `@public`"?
Fix is underway via #21418

Reply:
Yes, sorry, that is what I meant. I updated the statement above.